### DISCUSSION TITLE
Move tab from browser

### DISCUSSION THREAD

**Discussion by maciekdragan** [rxhanson/RectanglePro-Community#374] [Ideas]
I don't know if it's possible, but I often do the following:
- take one tab from browser (using chrome)
- move it to the right using rectangle
- click the browser
- move it to the left using rectangle

Would be great to automate this task, as I'm using it so often

**Comment by rxhanson** [comment]
Thanks for the feedback. It's been a while since I've looked into this, but last I did there was not a _good_ way to do it. If I figure out a good way to roll it in, then I'll add it in.

**Reply by maciekdragan** [reply]
Thank you, It would be great. Now there’s definitely too much clicking in the scenario.
